hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/ErrorsAndExceptionsEnricher.java

```diff
@@ -158,5 +167,18 @@ public void enrichTrace(StructuredTrace trace) {
               EnrichedSpanConstants.getValue(ErrorMetrics.ERROR_METRICS_TOTAL_SPANS_WITH_ERRORS),
               MetricValueCreator.create(errorCount));
     }
+
+    Map<String, AttributeValue> enrichedAttributes = new HashMap<>();
+
+    ApiTraceGraph apiTraceGraph = new ApiTraceGraph(trace);
+    for (ApiNode<Event> apiNode : apiTraceGraph.getApiNodeList()) {
+      Optional<Event> entryEvent = apiNode.getEntryApiBoundaryEvent();
+      int apiTraceErrorCount =
```

Comment:
Do we support enriching long attributes? If yes, I think we should not type case long to integer here